### PR TITLE
named witches page

### PR DESCRIPTION
**Pull request by rubyimrie**
adding flourish page, no big changes

### CODE REVIEW

File: pages/named-witches.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <line x1="12" y1="16" x2="12" y2="12"></line><line x1="12" y1="8" x2="12" y2="8"></line>
+          </svg>
+        </span>
+      </div>
+
+    </div>
+    <div style="height:90%; min-height: 400px;">
+      <iframe src='https://flo.uri.sh/story/1664928/embed' title='Interactive or visual content' class='flourish-embed-iframe' frameborder='0' scrolling='no' style='width:100%;height:100%;' sandbox='allow-same-origin allow-forms allow-scripts allow-downloads allow-popups allow-popups-to-escape-sandbox allow-top-navigation-by-user-activation'></iframe><div style='width:100%!;margin-top:4px!important;text-align:right!important;'><a class='flourish-credit' href='https://public.flourish.studio/story/1664928/?utm_source=embed&utm_campaign=story/1664928' target='_top' style='text-decoration:none!important'><img alt='Made with Flourish' src='https://public.flourish.studio/resources/made_with_flourish.svg' style='width:105px!important;height:16px!important;border:none!important;margin:0!important;'> </a></div>
```
Comment (Sephster) [inline pages/named-witches.vue:16]: Not sure where the text is coming from. I assume it is within Flourish itself but it says:

>  Press the arrows to see how this changes over time

or something similar. This confused me as there are arrows linking the named witches together so I tried clicking them. I also tried using my keyboard arrows.

I was wondering if we could update the text here to instead say something like.

> Use the navigational arrows to the left to...

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/named-witches.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/named-witches.vue:16]: I don't have access to the flourish but I will see if Ewan does.

Comment (rubyimrie) [reply pages/named-witches.vue:16]: Got the log in details and have changed the text.

Comment (Sephster) [reply pages/named-witches.vue:16]: Looks good to me. I will approve this but hold off merging until it's parent branch is also ready. Thanks Ruby

**Review by Sephster — APPROVED** [review]

**Pull request merged**